### PR TITLE
Conditionally display handshake filter

### PR DESCRIPTION
**Pull request by mjoyce91**
Only display the handshake filter if at least one bid season is selected.

Merge https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/725 first

### CODE REVIEW

File: src/Components/BidderPortfolio/BidControls/BidControls.jsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { checkFlag } from 'flags';
+import PreferenceWrapper from 'Containers/PreferenceWrapper';
 import SelectForm from '../../SelectForm';
-import { BID_PORTFOLIO_SORTS, BID_PORTFOLIO_FILTERS } from '../../../Constants/Sort';
+import { BID_PORTFOLIO_SORTS, BID_PORTFOLIO_FILTERS, BID_PORTFOLIO_SORTS_TYPE,
+  BID_PORTFOLIO_FILTERS_TYPE } from '../../../Constants/Sort';
```
Comment (elizabeth-jimenez) [inline src/Components/BidderPortfolio/BidControls/BidControls.jsx]: :) 'Constants/Sort'

File: src/Containers/BidderPortfolio/BidderPortfolio.jsx [hunk 2]
```diff
@@ -50,16 +53,20 @@ class BidderPortfolio extends Component {
   // Much of the logic is abstracted to a helper, but we need to set state within
   // the instance.
   onQueryParamUpdate(q) {
+    const { query, defaultPageNumber } = this.state;
```
Comment (elizabeth-jimenez) [inline src/Containers/BidderPortfolio/BidderPortfolio.jsx]: 🍰

File: src/Containers/BidderPortfolio/BidderPortfolio.jsx [hunk 3]
```diff
@@ -81,25 +88,34 @@ class BidderPortfolio extends Component {
       delete parsedQuery.type;
     }
     this.state.query.value = queryString.stringify(parsedQuery);
```
Comment (elizabeth-jimenez) [inline src/Containers/BidderPortfolio/BidderPortfolio.jsx]: i know this wasn't a recent change, but can be updated, if you want `query.value = queryString.stringify(parsedQuery);`

File: src/Containers/BidderPortfolio/BidderPortfolio.jsx [hunk 4]
```diff
@@ -81,25 +88,34 @@ class BidderPortfolio extends Component {
       delete parsedQuery.type;
```
Comment (elizabeth-jimenez) [inline src/Containers/BidderPortfolio/BidderPortfolio.jsx]: Not sure if im supposed to comment on just the changes? `omit(parsedQuery, ['type']);`

File: src/Containers/BidderPortfolio/BidderPortfolio.jsx [hunk 5]
```diff
@@ -81,25 +88,34 @@ class BidderPortfolio extends Component {
...
+    this.setState({ query });
   }
 
   // When we trigger a new search, we reset the page number and limit.
   createSearchQuery() {
+    const { defaultPageNumber, defaultPageSize, hasHandshake, ordering } = this.state;
     this.mapTypeToQuery();
     const query = {
-      page: this.state.defaultPageNumber.value,
-      limit: this.state.defaultPageSize.value,
```
Comment (elizabeth-jimenez) [inline src/Containers/BidderPortfolio/BidderPortfolio.jsx]: 🌻

**Comment by mjoyce91** [comment]
Good comments @elizabeth-jimenez - updated

**Pull request merged**